app/components/lead_providers/your_schools/table_row.html.erb

```diff
@@ -1,6 +1,6 @@
 <tr class="govuk-table__row">
-  <td class="govuk-table__cell" >
-    <%= govuk_link_to school.name, lead_providers_school_detail_path(school, selected_cohort: cohort.id) %>
+  <td class="school-name-cell govuk-table__cell" >
+    <%= govuk_link_to school.name, lead_providers_partnership_path(partnership) %>
```

Comment:
What does `school-name-cell` do? Is the css for it gonna come later?

Reply:
Yup, that should be part of another PR. Will remove for now